src/snowflake/cli/app/snow_connector.py

```diff
@@ -157,8 +157,9 @@ def _update_connection_application_name(connection_parameters: Dict):
     """Update version and name of app handling connection."""
     connection_application_params = {
         "application_name": PARAM_APPLICATION_NAME,
-        "_internal_application_name": PARAM_INTERNAL_APPLICATION_NAME,
-        "_internal_application_version": VERSION,
+        "application": PARAM_APPLICATION_NAME,
+        "internal_application_name": PARAM_INTERNAL_APPLICATION_NAME,
+        "internal_application_version": VERSION,
```

Comment:
For some reason/convention unknown to me these key names started with underscore in SQLAlchemy.

Reply:
Not sure 
https://github.com/snowflakedb/snowflake-sqlalchemy/blob/58fb1bd556cddbd2b7cf2b8ccebf0ead3363e113/src/snowflake/sqlalchemy/_constants.py#L7-L9

Reply:
yupp, your'e right 
https://github.com/snowflakedb/snowflake-sqlalchemy/blob/58fb1bd556cddbd2b7cf2b8ccebf0ead3363e113/src/snowflake/sqlalchemy/util.py#L108